templates/zerver/help/add-custom-emoji.md

```diff
@@ -1,7 +1,8 @@
 # Add custom emoji
 
-If you are an administrator of a Zulip organization, you can add
-custom emojis to your organization for other members to use.
+{!admin-only.md!}
```

Comment:
There's actually a setting which controls whether normal users or only admins can change this.  We probably can't use this macro here, but should instead link to said setting.